zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -397,14 +417,57 @@ impl<'a, F: Field> EVMConstraintBuilder<'a, F> {
         self.query_cell_with_type(CellType::LookupByte)
     }
 
-    pub(crate) fn query_word_rlc<const N: usize>(&mut self) -> RandomLinearCombination<F, N> {
-        RandomLinearCombination::<F, N>::new(self.query_bytes(), self.challenges.evm_word())
+    // TODO remove me
+    // pub(crate) fn query_word_rlc<const N: usize>(&mut self) -> RandomLinearCombination<F, N> {
+    //     RandomLinearCombination::<F, N>::new(self.query_bytes(), self.challenges.evm_word())
+    // }
+
+    // default query_word is 2 limbs. Each limb is not guaranteed to be 128 bits.
+    pub fn query_word_unchecked<const N: usize, const N2: usize>(&mut self) -> Word<Cell<F>> {
```

Comment:
Please document what is `N` and `N2` in all the query methods that use these associated consts.